Sources/SwiftOgg/OGGConverter.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2022 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import AVFAudio
+import AVFoundation
+
+public enum OGGConverterError: Error {
```

Comment:
It would be nice if this would only expose its own errors instead of forwarding ones from undearneath e.g. `OpusError.invalidState`, `OggError.internalError`, even if only wrapping them.